sys-apps/miller/miller-5.2.2.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools
+
+DESCRIPTION="A tool like sed, awk, cut, join, and sort for name-indexed data (CSV, JSON, ..)"
```

Comment:
Please group the variables alike in the [devmanual examples](https://devmanual.gentoo.org/quickstart/) (look at the later examples).